qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -563,45 +567,56 @@ def generate_hamiltonian(
     return convert_observable(h_of), nr_qubits
 
 
-def sd_excitations(n_electrons, n_orbitals, delta_sz=0):
-    r"""Generates single and double excitations from a Hartree-Fock (HF) reference state.
+def excitations(n_electrons, n_spin_orbitals, delta_sz=0):
+    r"""Generates single and double excitations from a Hartree-Fock reference state.
 
-    The singly- and doubly-excited configurations are generated by acting with the operators
-    :math:`\hat T_1` and :math:`\hat T_2` on the HF state:
+    Single and double excitations can be generated by acting with the excitation operators
+    :math:`\hat T_1` and :math:`\hat T_2` on the Hartree-Fock (HF) reference state:
 
     .. math:
```

Comment:
```suggestion
    .. math::
```
It wasn't showing up for me without this

Reply:
Thanks @trbromley !